tests/admin/applications/test_index.py

```diff
@@ -0,0 +1,62 @@
+class TestAdminElements:
+
+    def test_homepage_elements(self, app, client):
+        with app.app_context():
+            with client.session_transaction() as session:
+                session['signedIn'] = 'test.email@example.com'
+
+            response = client.get('/applications')
+            html = response.data.decode()
+            assert response.status_code == 200
+            assert 'View and download GRC applications' in html
+            assert 'Download GRC applications' in html
+            assert 'New applications' in html
+            assert 'Downloaded applications' in html
+            assert 'Completed applications' in html
+
+    def test_new_applications_present(self, app, client, submitted_application):
+        with app.app_context():
+            with client.session_transaction() as session:
+                session['signedIn'] = 'test.email@example.com'
+
+            response = client.get('/applications#new')
+            html = response.data.decode()
+            assert response.status_code == 200
+            assert 'New applications' in html
+            assert 'View application' in html
+            assert 'Reference number' in html
+            assert 'Applicant name' in html
+            assert 'Submitted' in html
+            assert f'/applications/{submitted_application.reference_number}' in html
```

Comment:
I think a good test would be to assert that the application data is present and correct in the headers. E.g. Headers in new applications is Reference number, applicant name and Submitted date and assert these are correct